packages/rum-core/src/common/utils.js

```diff
@@ -149,6 +149,8 @@ function checkSameOrigin(source, target) {
   let isSame = false
   if (typeof target === 'string') {
     isSame = source === target
+  } else if (typeoftarget && typeof target.test === 'function') {
```

Comment:
`target && `

Reply:
Oh my godddddd I can't believe it!!
Sorry for your time 😞

Reply:
But I see you found the bug related to NPM7, now I hope I will get the git hooks and tests